build-utils/run_ci.sh

```diff
@@ -23,9 +24,13 @@ if [[ -z $COMMAND || -z $STUDY_KEY || -z $BRANCH || ($COMMAND == "deploy" && -z
   echo "    run-tests"
   echo "        Run tests for given study and given branch"
   echo "    run-e2e-tests"
-  echo "        Run Playwright E2E tests for given branch against specified TARGET_ENV (dev, test or staging)"
-  echo "        STUDY_KEY is unused. CI API call will run all tests"
-  echo "        Example: ./run_ci.sh run-e2e-tests singular pepper-227-test dev"
+  echo "        Run Playwright E2E tests on develop branch against specified TARGET_ENV (dev or test)"
+  echo "        STUDY_KEY: Run this study E2E test suite"
+  echo "        BRANCH: develop"
+  echo "        TARGET_ENV: dev or test"
+  echo "        E2E_TEST_PARALLELISM: Set CircleCi parallelism. Default value is 1"
+  echo "        Example 1 (run all tests against Dev env): ./run_ci.sh run-e2e-tests UNKNOWN develop dev 2"
+  echo "        Example 2 (run Singular test suite against Dev env): ./run_ci.sh run-e2e-tests singular develop dev"
```

Comment:
Updated cmd that triggers `playwright-e2e-test-workflow` workflow.

Usage screenshot:

<img width="777" alt="Screenshot 2023-01-25 at 4 53 24 PM" src="https://user-images.githubusercontent.com/35533885/214700121-448b44e3-98a3-4cf4-b201-29a456450a73.png">